### DISCUSSION TITLE
Sub-Sections

### DISCUSSION THREAD

**Discussion by Realium3D** [Mustard2/MustardUI#87] [Ideas]
I would love to see the ability to add collapsable Sub-Sections to a collapsable custom properties section. For example, I have a lot of custom shape keys for various parts of the body and shader controls, it would be nice to be able to organise them into something like this:

- Body Shapes
      - Face
      - Torso
      - Arms
      - Etc

- Skin Settings
      - Face
      - Torso
      - Nails
      - Etc

**Comment by Mustard2** [comment]
Interesting idea!

I was wondering how it can be implemented from a UI point of view. Atm you have the list of sections, I have to think how to add the subsection part there, I don't want to make the UI too complex. If you have any idea, I'm obviously all ears. :)

**Reply by Realium3D** [reply]
It's a little outside my knowledge base sadly, but I was toying with the idea of duplicating how sections are made, calling them subsections, and then having it so that a subsection can be assigned a "parent" section. Similar to how we can add custom properties to a section.

**Reply by Mustard2** [reply]
Yeah, from a UI point of view that would be great. I have to think about how to implement it technically though :D

**Comment by Mustard2** [comment]
Opened an issue to keep track of this ad developement level: #88
